src/smolvm/ssh.py

```
@@ -61,6 +61,13 @@
 #     logging.getLogger("paramiko.transport").setLevel(logging.WARNING)
 logging.getLogger("paramiko.transport").setLevel(logging.CRITICAL)
 
+# Poll backoff for wait_for_ssh: start tight (20ms) so we notice sshd within
+# tens of ms of it coming up early in boot, grow ×1.5 per miss, and let the
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)